### ISSUE TITLE
No new versions for MacOS

### ISSUE THREAD

**Issue by borondics** [cerebroapp/cerebro#552]
- [ ] I am on the [latest](https://github.com/cerebroapp/cerebro/releases/latest) Cerebro.app version
- [x] I have searched the [issues](https://github.com/cerebroapp/cerebro/issues) of this repo and believe that this is not a duplicate

- **OS version and name**:  12.2.1
- **Cerebro.app version**: 
- **Relevant information from devtools** _(See above how to open it)_: 

## Issue

Will the new version (0.5 at the moment) be available as a DMG or HomeBrew installer?

**Comment by 2br-2b** [comment]
Same with on Windows. Version 0.3.1 worked fine, but 0.3.2 introduced some breaking changes. After the app auto-updates (which can't be disabled), I get three error popups on launching the app, my apps don't show up when searching, and the icon isn't appearing in the taskbar.

**Comment by dubisdev** [comment]
Hi all

Current development team doesn't have any MacOS computer so we can't test the app.

This doesn't mean Cerebro can't run on MacOS, but we don't want to release to an untested platform.

An executable can be created by following this steps:

1. Clone the repo - Using `git clone` or the GH client
2. Install dependencies - `yarn --force`
3. Package the app - `yarn package`
4. The executables are released to the `releases` folder.

**Comment by borondics** [comment]
Thanks for the info!

I packaged it on Mac and it is not working unfortunately, although everything seems fine but it doesn't produce windows at all. I'd be happy to try Cerebro out in the future!

**Comment by jessebot** [comment]
Popping in to say that after following the instructions above from @dubisdev , Cerebro _does_ work on macOS 13.0 Ventura as of my test today with Cerebro-0.8.0. After you run `yarn package` as mentioned, you will get a releases folder. `cd` into that directory, and then run open on the generated `dmg` file. So in my case it was:

```bash
git clone git@github.com:cerebroapp/cerebro.git
brew install yarn
yarn --force
yarn package
cd releases
open Cerebro-0.8.0.dmg
```

Then it opens this window:
<img width="529" alt="Screenshot 2022-11-09 at 10 16 01" src="https://user-images.githubusercontent.com/2389292/200789791-17defd9a-dc46-4127-95cc-c6561e84a642.png">

Drag the Cerebro app into the applications folder, and then it should work as a normal.

Hope that helps anyone stuck :)

**Comment by jessebot** [comment]
@dubisdev it might make sense to add a section for macOS, but explicitly say it's untested, and that you're looking for people to verify it. I'd be happy to help out on this project by coming in to test it on macOS, and maintaining a section of the docs for macOS.

Right now, the current official cerebro website listed on your github has a download link for macOS that downloads version 0.3.1, which crashes on launch on macOS Ventura with a javascript error, because I believe it's missing something, because after that failed, I did the above mentioned methods, and then deleted cerebro-0.8.0 from my applications folder, to try the 0.3.1 download from the website again, and it worked.

I'd like to help document some of this, because it drained an hour of my time today, and it would be nice to help others avoid that, because this project is neat ❤️ Please feel free to tag me  on issues if you need help testing things for macOS as well. My primary operating systems are macOS and Debian.

**Comment by dubisdev** [comment]
Hi @jessebot and thanks for your help. We recently fixed the bug that was causing Cerebro to fail in MacOS (https://github.com/cerebroapp/cerebro/pull/618) thanks to some new contributors (https://github.com/cerebroapp/cerebro/issues/602)

That PR also includes adding macOS to the CI build workflow, so in next releases a MacOS executable will be released 🙌 and will be available in the website.

Of course, feel free to open as PRs as you want to improve the documentation (none of the current developers are English native speakers and the docs might have grammar errors 🙄)

I'm conscient of the Cerebro website issue. We don't have access to the old domain as it was bought by the previous project maintainer. We migrated the website to NextJS and is now deployed to [cerebroapp.vercel.app](https://cerebroapp.vercel.app) with automatically updated download links.

Thanks again for your interest in the project and welcome aboard! 💫
